data/accounts/fr_CH/acctchrt_pme-19.gnucash-xea

```diff
@@ -0,0 +1,2382 @@
+    <act:type>EQUITY</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>2950</act:code>
+    <act:description>2950</act:description>
+    <act:parent type="guid">35e6318959294d84994376047bef94a7</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Réserves facultatives</act:name>
+    <act:id type="guid">6fa91e39a5f7488cb7d04fc6c2a56810</act:id>
+    <act:type>EQUITY</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>2960</act:code>
+    <act:description>2960</act:description>
+    <act:parent type="guid">35e6318959294d84994376047bef94a7</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Bénéfice ou perte reporté</act:name>
+    <act:id type="guid">780726e389184e1ab7f7fbfe16d5f17b</act:id>
+    <act:type>EQUITY</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>2970</act:code>
+    <act:description>2970</act:description>
+    <act:parent type="guid">35e6318959294d84994376047bef94a7</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Bénéfice ou perte de l'exercice</act:name>
+    <act:id type="guid">9bb074b99e4d4a40b9151c2289c10073</act:id>
+    <act:type>EQUITY</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>2979</act:code>
+    <act:description>2979</act:description>
+    <act:parent type="guid">35e6318959294d84994376047bef94a7</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Propres actions, parts sociales, droits de participations (poste négatif)</act:name>
+    <act:id type="guid">4df7a3d99e0e407796920758f2708f7d</act:id>
+    <act:type>EQUITY</act:type>
+    <act:commodity>
+    <cmdty:space>CURRENCY</cmdty:space>
+    <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:code>2980</act:code>
+    <act:description>2980</act:description>
+    <act:parent type="guid">35e6318959294d84994376047bef94a7</act:parent>
+  </gnc:account>
+  <gnc:account version="2.0.0">
+    <act:name>Soldes initiaux</act:name>
+    <act:id type="new">a19c505d4b414bf3f5b574104959f534</act:id>
+    <act:type>EQUITY</act:type>
+    <act:commodity>
+      <cmdty:space>ISO4217</cmdty:space>
+      <cmdty:id>CHF</cmdty:id>
+    </act:commodity>
+    <act:commodity-scu>100</act:commodity-scu>
+    <act:description>Soldes initiaux</act:description>
+    <act:parent type="new">47469471b603439c88189eca6c2e86e1</act:parent>
```

Comment:
From my understanding it belongs under "Capitaux propres", which you named `Capitaux propres (personnes morales)`.
Is `personnes morales` really part of the name or would it better fit into 
`<act:description>Personnes morales</act:description>`?

I just saw, we didn't mention `<act:description>` and added the `List of useful columns` there.

Reply:
OK for me, thanks for the suggestion.